python/test/regression/test_cast_matmul.py

```diff
@@ -31,13 +31,25 @@
 
 
 @triton.jit
-def matmul_kernel(A, B, C, M, N, K,  #
-                  stride_am, stride_ak,  #
-                  stride_bk, stride_bn,  #
-                  stride_cm, stride_cn,  #
-                  dot_out_dtype: tl.constexpr,  #
-                  BLOCK_M: tl.constexpr, BLOCK_N: tl.constexpr,  #
-                  BLOCK_K: tl.constexpr, GROUP_M: tl.constexpr):
+def matmul_kernel(
+    A,
+    B,
+    C,
+    M,
+    N,
+    K,  #
+    stride_am,
+    stride_ak,  #
+    stride_bk,
+    stride_bn,  #
+    stride_cm,
+    stride_cn,  #
+    dot_out_dtype: tl.constexpr,  #
+    BLOCK_M: tl.constexpr,
+    BLOCK_N: tl.constexpr,  #
+    BLOCK_K: tl.constexpr,
+    GROUP_M: tl.constexpr,
```

Comment:
I completely understand why black chose this style, but man it really makes functions with long argument lists verbose and currently that's pretty much all non-trivial triton kernels.

Reply:
Yeah, this is a fair critique. I'm a fan of the verbosity personally, but I definitely understand why one might not be